02-starting-with-data.Rmd

```diff
@@ -278,17 +303,23 @@ or decreasing order, test `1:10` and `10:1` for instance.
 You can also exclude certain indices of a data frame using the "`-`" sign:
 
 ```{r, purl=FALSE}
-surveys[, -1]          # The whole data frame, except the first column
+surveys[, -1]         # The whole data frame, except the first column
 surveys[-(7:34786), ] # Equivalent to head(surveys)
```

Comment:
...or `tail(34779)`, I suppose  :)

Reply:
I tried my best to keep to a minimal rewrite :joy: 

If anything, it almost makes sense to move this one a little later in the lesson so that we can subset using `surveys[-7:nrow(surveys), ]` rather than hard-code the number of rows in the table, which feels a _little_ bit like teaching people to use "magic numbers" instead of semantic functions or variables. But this feels like a less clear case than the rest of the PR, so I just left it as it was :joy:

Reply:
Very good point about not using "magic numbers". I believe `nrow()` is mentioned earlier in this lesson (`inspecting data frames`), so it would be ok to use it here, wouldn't it?

I know you want to keep the rewrite minimal, but from my perspective using `nrow()` instead of `34786` is much better. And since you are already improving this section, you could just fix this along the way. 
 @fmichonneau , what do you think?